### ISSUE TITLE
[dev/xml] Fix XSL scripts compilation in XmlCompiledTransform

### ISSUE THREAD

**Issue by sepidehMS** [dotnet/corefx#9873]
If XSL contains scripts (C#, VB, JScript), the script is currently not being compiled from a dll correctly. Either need to bring back CodeDom or find a workaround.

The tests related to this scenario should be re-enabled. (Look for ActiveIssue(9873))

This needs to be done as a part of dev/xml branch work

**Comment by danmosemsft** [comment]
Minor work as soon as Code DOM is present.

**Comment by danmosemsft** [comment]
CodeDOM not currently planned for 1.2

**Comment by stephentoub** [comment]
#12117 adds System.CodeDom, but without compilation support, e.g. you can build up an object model and generate C#/VB from it, but you'll get PlatformNotSupportedException currently if you try to then compile that source or object model into an assembly.

**Comment by weshaggard** [comment]
How deep of a dependency is this? Can we break-out these APIs into extensions or something else? I would really like to break the XML(part of netstandard) dependency on CodeDom(not part of netstandard).

**Comment by danmosemsft** [comment]
@weshaggard How are you expecting extensions to work -- would XML call CodeDOM via reflection?

Either way -- I think that is out of scope for this effort, and it seems to me the right thing is for @sepidehMS to remove the reference, and #ifdef0--out the calls to CodeDOM. Do you agree?

**Comment by weshaggard** [comment]
Before we can decide what to do we need to understand what breaks by removing this dependency. If it is isolated then we can perhaps move some of the impacted APIs from netstandard into an extension and thus can be implemented on top of netstandard and be opt-in.

**Comment by Petermarcu** [comment]
Is this in netstandard2.0 per the label? If so, please put it in the 1.2 milestone.

**Comment by karelz** [comment]
Won't fix for now.

**Comment by weshaggard** [comment]
@karelz while I'm fine with not actually fixing the compilation scenario I would like to better understand the dependency better because it might end up identifying some APIs we need to remove from netstandard before it ships.
